scripts/install-dev.sh

```diff
@@ -119,7 +119,21 @@ install_slack_cli() {
         #
 
         if [ "$(uname)" == "Darwin" ]; then
+            if version_lt "$SLACK_CLI_DEV_VERSION" "3.3.0"; then
                 slack_cli_url="https://downloads.slack-edge.com/slack-cli/slack_cli_${SLACK_CLI_DEV_VERSION}_macOS_64-bit.tar.gz"
+            else
+                case "$(uname -m)" in
+                    x86_64)
+                        slack_cli_url="https://downloads.slack-edge.com/slack-cli/slack_cli_${SLACK_CLI_DEV_VERSION}_macOS_amd64.tar.gz"
+                        ;;
+                    arm64 | aarch64)
+                        slack_cli_url="https://downloads.slack-edge.com/slack-cli/slack_cli_${SLACK_CLI_DEV_VERSION}_macOS_arm64.tar.gz"
+                        ;;
+                    *)
+                        slack_cli_url="https://downloads.slack-edge.com/slack-cli/slack_cli_${SLACK_CLI_DEV_VERSION}_macOS_64-bit.tar.gz"
+                        ;;
+                esac
+            fi
```

Comment:
📣 We're guarding against specific installations of versions prior to next release here!

This adds some overhead to this script, but I don't think backporting releases can be done without so much effort and potential error otherwise...